src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -123,6 +123,10 @@ List<List<ChannelData<T>>> getData() {
   // blob encoding version
   private final Constants.BdecVersion bdecVersion;
 
+  // Indicates if it's flushing to Iceberg tables, a blob could only contain one chunk under Iceberg
+  // mode
+  private final boolean isIcebergMode;
```

Comment:
Can this have a more general name, e.g. `isNonInterleavedMode`?

Reply:
Yeah, we can make this more general for the internal names since we disable interleaved mode due to other reasons in the future, not only Iceberg

Reply:
Removed the non-interleave parameter for now as we could control this via `MAX_CHUNKS_IN_BLOB_AND_REGISTRATION_REQUEST`